### ISSUE TITLE
Regarding the issue of including negative samples in the training set

### ISSUE THREAD

**Issue by jianjun0407** [MIC-DKFZ/nnUNet#1412]
First of all, thank you for sharing. I am now doing a tumor segmentation task, which is a 2 classification task. 0 represents the background and 1 represents the tumor region.
Now it turns out that the false positives are very high, in other words, testing the data of healthy people, there are also some false positive areas on the prediction
Now I'm going to add some negative samples to the training set, which is the data of healthy people. During training, the program kept saying "case dose not contain any foregroud class"
Please ask:
(1) When the program makes such prompts, does it automatically exclude such data during training? In other words, these negative samples were not involved in the training？
(2) How to add negative samples with all 0 labels to the training data?
![捕获](https://user-images.githubusercontent.com/19251556/233267054-d09d733b-7d0d-469e-b51e-fdfb68827ba5.PNG)

**Comment by FabianIsensee** [comment]
Hey, you can ignore these prompts. They have been removed in nnunet v2 because they are only confusing people. The way you did it is correct: just add these images to your training set with empty segmentation masks

**Comment by provbs** [comment]
just what I was wondering!

thank you for the question and the answer!

**Comment by hj950815** [comment]
Hi, when i change oversample_foreground_percent from 0.33 to 0.7, fewer false positives, but more false negatives.It is difficult to reduce false positives while maintaining sensitivity
Is the approach proposed here a more efficient approach? Add negative images to training set with empty segmentation masks
Can you help me?
